framework/src/imgui_manager.cpp

```diff
@@ -292,12 +312,31 @@ namespace gvk
 		ImGui::Render();
 		auto cmdBfr = mCommandPool->alloc_command_buffer(vk::CommandBufferUsageFlagBits::eOneTimeSubmit); 
 		cmdBfr->begin_recording();
+
+		// if no invokee has written on the attachment (no previous render calls this frame),
+		// reset layout (cannot be "store_in_presentable_format").	
+		if (aFramePreviousRenderCallsCount == 0) {			
```

Comment:
First of all see comment below (line 331), then continue here:

We could just use `if (!mainWnd->has_consumed_current_image_available_semaphore())` also here, couldn't we?!

Reply:
changes applied